daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -211,10 +211,12 @@ private[lf] object Speedy {
   private[speedy] case class DisclosureTable(
       contractIdByKey: Map[crypto.Hash, SValue.SContractId],
       contractById: Map[SValue.SContractId, (TypeConName, SValue)],
+      disclosedContractById: Map[V.ContractId, DisclosedContract],
```

Comment:
Partial transactions store the disclosed contracts that are used. in order to facilitate contract ID and key to disclosed contract lookups (when evaluating `SBFetchAny`, `SBUFetchKey` and `SBULookupKey`), we add extra "indexing" maps to the disclosure table.